cranelift/control/src/chaos.rs

```diff
@@ -1,28 +1,66 @@
+use std::fmt::Display;
+
 use arbitrary::Arbitrary;
 
 /// The control plane of chaos mode.
 /// Please see the [crate-level documentation](crate).
 #[derive(Debug, Clone, Default)]
 pub struct ControlPlane {
     data: Vec<bool>,
+    fuel: Option<u32>,
 }
 
 impl Arbitrary<'_> for ControlPlane {
     fn arbitrary<'a>(u: &mut arbitrary::Unstructured<'a>) -> arbitrary::Result<Self> {
         Ok(Self {
             data: u.arbitrary()?,
+            fuel: None,
         })
     }
 }
 
 impl ControlPlane {
+    /// Returns `true` if fuel wasn't activated in the first place, `false`
```

Comment:
This doc-comment covers two of the three cases I think -- it also returns `true` if fuel is activated and there is enough fuel, right?

It might make sense to describe at a slightly higher level: "Tries to consume fuel, returning true if successful (or if fuel-limiting is disabled)."